python/lsst/shapelet/tractor.py

```diff
@@ -57,8 +65,11 @@ def registerRadialProfiles():
         except lsst.pex.exceptions.Exception:
             warnings.warn("No C++ profile for multi-Gaussian pickle file '%s'" % filename)
             continue
-        with open(os.path.join(dataDir, filename), 'r') as stream:
-            array = pickle.load(stream)
+        with open(os.path.join(dataDir, filename), 'rb') as stream:
+            if sys.version_info[0] >= 3:
+                array = pickle.load(stream, encoding='latin1')
```

Comment:
The pickle files are in latin1???? Is this because it's trying to read pickle files that were written on Python 2 without being in binary mode?